rubicon_ml/viz/dataframe_plot.py

```diff
@@ -115,6 +124,8 @@ def load_experiment_data(self):
             self.plotting_func_kwargs["color_discrete_sequence"] = get_rubicon_colorscale(
                 len(self.experiments),
             )
+        if self.data_df is None:
+            raise Exception(f"No dataframe with name {self.dataframe_name} found!")
```

Comment:
Should this be RubiconException?

Reply:
I am not sure I just tried this and ran into some errors with the testing.

Reply:
See my comment below - we will need to test for the same exception we raise here. The typical pattern is to raise one of the [concrete exceptions](https://docs.python.org/3/library/exceptions.html#concrete-exceptions) or an exception that we control (like the Rubicon Exception). Since we mostly raise the `RubiconException` throughout I think we should do so here.

Reply:
I tried raising and asserting a `RubiconException` however this is failing the test. I am not sure why because when I also try this in the notebook, the `RubiconException` is being raised.